doc/api/Module.md

```diff
@@ -0,0 +1,186 @@
+}
+```
+
+<a name="matchd"></a>
+## ~~match(text, commandPrefix) ⇒ <code>boolean</code>~~
+__Deprecated__  
+Use [match(event, commandPrefix)](#match) instead.
+
+Evaluates if the module should be run for the given message.  
+This will be run for every received message so should not contain any time consuming code, instead should just return true/false depending on if it should be run or not.  
+
+**Note**: This method should not be implemented if the `"command"` property has been provided in `kassy.json`. See [Kassy.json.md#command](./Kassy.json.md#command) for details.  
+**Node**: This method will be called only if the first parameter is called `text`.  
+**Kind**: API method  
+**Returns**: <code>true</code> if the module should run, false otherwise.
+
+| Param | Type | Description |
+| --- | --- | --- |
+| text | <code>string</code> | An string containing the body of the message. See [Event.md#body](./Event.md#body). |
+| commandPrefix | <code>string</code> | The command prefix associated with the integration that the event was raised on. See [Api.md#commandPrefix](./Api.md#commandPrefix). |
+
+**Example**  
+A simple hello world implementation:
+```js
+exports.match = (text, commandPrefix) => {
+    return text.startsWith(commandPrefix + 'HelloWorld');
+}
+```
+
+<a name="help"></a>
+## help(commandPrefix) ⇒ <code>array[array[string, string, string?]]</code>
+Gets help text for the module.
+
+**Optional**  
+**Note**: This method should not be implemented if the `"help"` property has been provided in `kassy.json`. See [Kassy.json.md#help](./Kassy.json.md#help) for details.  
+**Kind**: API method  
+**Returns**: <code>array[array[string, string, string?]]</code>, see [Kassy.json.md#help](./Kassy.json.md#help) for details.
+
+| Param | Type | Description |
+| --- | --- | --- |
+| commandPrefix | <code>string</code> | The command prefix associated with the integration that the help was requested on. See [Api.md#commandPrefix](./Api.md#commandPrefix). |
+
+**Example**  
+A simple hello world implementation:
+```js
+exports.help = (commandPrefix) => {
+    return [
+        [commandPrefix + 'HelloWorld', 'Prints "Hello World".'],
+        [commandPrefix + 'HelloWorldExtended', 'Prints "Hello World".', 'Prints "Hello World" with extended help.']
+    ];
+}
+```
+
+<a name="load"></a>
+## load()
+Called when the all the resources for a module are loaded. Properties such as `config` may not exist until this method is called.
+
+**Optional**  
+**Kind**: API method  
+
+<a name="unload"></a>
+## unload()
+When the module resources should be unloaded and timers should be cancelled - this is usually so your module can be updated or unloaded.
+
+**Optional**  
+**Kind**: API method  
+
+<a name="config"></a>
+## config ⇒ <code>Object</code>
+**Kind**: Object property
+
+Object for storing configuration data of the module within. Any property set on this object will automatically be persisted between restarts of your module.
```

Comment:
Perhaps we can elaborate further that this config is different from the global `config.json`